src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpCertificateHelper.cs

```diff
@@ -1,16 +1,17 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
 using System.Net.Security;
 using System.Runtime.InteropServices;
 using System.Security.Cryptography.X509Certificates;
 
 namespace System.Net.Http
 {
-    // TODO: Move/rename to Common/System/Net as part of System.Private.Networking.dll refactor.
-    internal static class CertificateHelper
+    internal static class WinHttpCertificateHelper
     {
+        private const string ClientAuthenticationOID = "1.3.6.1.5.5.7.3.2";
+        
+        // TODO: Merge with System.Net.Security certmodule.cs and move/rename to Common/src//System/Net.
```

Comment:
The file did exist a few days ago until the most recent PAL layer stuff was written. I will revise comment.